package.json

```diff
@@ -19,7 +19,7 @@
     "distance transform"
   ],
   "author": "Vladimir Agafonkin",
-  "license": "ISC",
+  "license": "BSD",
```

Comment:
I believe the correct one for the text is `BSD-2-Clause`: https://spdx.org/licenses/

Reply:
Fixed